setup.py

```diff
@@ -105,6 +105,7 @@ def txt_read(*paths):
         "GeoNodes": ["cartopy[plotting]", ],  # required for visualization of GeoArray and GeoSurface nodes
         "BAG": ["hydroffice.bag>=0.2.10", ],  # required by BAG plugin
         "OpenDAP": ["pydap", ],  # required by OpenDAP plugin
+        "ADIOS": ["adios>=1.9.1b19", ],  # required by ADIOS plugin
```

Comment:
**On Windows**, I believe that it would be better that `pip install hdf-compass[adios]` gives back the warning: `hdf-compass does not provide the extra 'adios'` rather than failing in the following way:

```
Collecting adios>=1.9.1b19 (from hdf-compass==0.6.1b1)
  Downloading adios-1.9.1b19.tar.gz (199kB)
    100% |################################| 204kB 2.4MB/s
    Complete output from command python setup.py egg_info:
    adios_config is not installed nor found. Please install Adios or check PATH.
```